src/components/DetailsMenu/DetailsMenuCard/ButtonRow/index.tsx

```diff
@@ -114,17 +114,6 @@ const ButtonRow: React.FC = () => {
           />
         }
       >
-        <MenuItem
-          as="a"
-          data-cy="lobster-button"
-          disabled={!lobsterURL}
-          glyph={<Icon glyph="Export" />}
-          href={lobsterURL || ""}
-          onClick={() => sendEvent({ name: "Opened Lobster Logs" })}
-          target="_blank"
-        >
-          Lobster
-        </MenuItem>
         {legacyJobLogsURL && (
```

Comment:
If there's no job logs, the menu now can be empty and look weird for non-resmoke logs:
<img width="524" alt="Screenshot 2024-01-26 at 10 17 04 AM" src="https://github.com/evergreen-ci/parsley/assets/47064971/ed04f171-9499-47a9-a45c-33526af8c8f1">

I think the easiest way to change this would be to move `legacyJobLogsURL` and wrap it as a conditional around the whole` <Menu />` or disable the `<MenuItem />`